selenium_tests/lti_emailer/test_setup.py

```diff
@@ -0,0 +1,41 @@
+__author__ = 'hic048'
+import unittest
+from ddt import ddt, data, unpack
+
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium.webdriver.support.ui import Select
+from selenium_tests.lti_emailer.page_objects.add_user_po import UserAdd, Locator3
+from selenium_tests.lti_emailer.emailer_base_test_case import CANVAS_ROLES
+from selenium_tests.base_test_case import get_xl_data
+
+# BASE_URL = /courses/6389/external_tools/1759
+#URL = "http://canvas.icommons.harvard.edu/courses/6389/users"
+
+@ddt
+class Addingusers(EmailerBaseTestCase):
```

Comment:
Is this used anywhere? If not, can it be removed? If it's meant to be used in a future test, you can add it back in then.